io.github.qnapi.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.qnapi
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "5.14"
+command: qnapi
+rename-icon: qnapi
+rename-desktop-file: qnapi.desktop
+copy-icon: true
+finish-args:
```

Comment:
Needs `--own-name='org.kde.*'` for tray icon to work.

Reply:
For me, on KDE tray icon is working, but surely I can add that.